src/modules/75-cf/strings/strings.en.yaml

```
@@ -610,9 +610,13 @@ onboarding:
   oneCreateAFlag: '1. Create a Flag'
   setUpApp: '2. Set Up Your Application'
   testYourFlag: '3. Test Your Flag'
-  letsStart: Let's start off by creating a simple flag
-  inputLabel: Give your flag a memorable name
-  successLabel: 'You have created <strong>{{name}}</strong> with Id: <strong>{{identifier}}</strong>'
+  letsGetStarted: Let's get started
+  createFlag: Create a Feature Flag
+  createFlagDescription: A feature flag controls what you want to release, who receives it, and how it will be delivered.
```

It'll be good to capitalise these Harness nouns for consistency
```suggestion
  createFlagDescription: A Feature Flag controls what you want to release, who receives it, and how it will be delivered.
```

---

+1

---

Done